test/benchmarks/micro/icall_hash.c

```diff
@@ -0,0 +1,27 @@
+#include <inttypes.h>
+
+#define WASM_EXPORT __attribute__((visibility("default")))
+
+uint64_t fnv1(uint64_t state, uint64_t input)
+{
+    return (state ^ input) * 0x100000001b3;
+}
+
+uint64_t threeab(uint64_t state, uint64_t input)
+{
+    return (3 * state) + input;
+}
+
+typedef uint64_t (*hashfn)(uint64_t state, uint64_t input) ;
+
+static hashfn fns[] = { fnv1, threeab };
```

Comment:
Could also just use `{ fnv1, fnv1 };` if we want to reduce the surface area.

Reply:
This is fine.

Did you format the code?

Reply:
With clang-format? No, will do.